packages/interfaces/src/index.ts

```diff
@@ -50,6 +51,22 @@ export interface LightPush extends PointToPointProtocol {
   ) => Promise<SendResult>;
 }
 
+export interface PeerExchange extends PointToPointProtocol {
+  query(params: PeerExchangeQueryParams): Promise<PeerExchangeResponse>;
+}
+
+export interface PeerExchangeQueryParams {
+  numPeers: bigint;
+}
+
+export interface PeerExchangeResponse {
+  peerInfos: PeerInfo[];
+}
+
+export interface PeerInfo {
+  ENR?: Uint8Array;
```

Comment:
This is not really useful per se. We do have a `ENR` class that could/should be used

Reply:
Fair to merge with byte array and do follow up PR to provide `ENR` interface.